File: 成绩分析.py
n =int(input())
max = -1
min = 101
sum = 0.
for i in range(n):
    s = int(input())
    sum = sum+s
    if max<s:max = s
    if min>s:min = s
print("{}\n{}\n{:.2f}".format(max,min,sum/n))

File: 成绩分析.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    int max = -1, min = 101;
    double sum = 0;
    cin >> n;
    for (int i = 0; i < n; i++) {
        int s;
        cin >> s;
        sum += s;
        if (max < s)max = s;
        if (min > s)min = s;
    }

    printf("%d\n%d\n%.2f",max,min,sum/n );

    return 0;
}

